test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -270,17 +282,15 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ;
       ;; (ie, this does not throw an exception)
       (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-url]))))
 
-(deftest chain-files-test
-  (testing "chain-files flattens files correctly"
-    (let [bootstrap-one (StringReader. "one\ntwo")
-          bootstrap-two (StringReader. "three\nfour")]
-      (is (=
-           ["one", "two", "three", "four"]
-           (chain-files [bootstrap-one bootstrap-two])))))
-  (testing "chain-files removes empty files"
-    (let [bootstrap-one (StringReader. "one\ntwo")
-          bootstrap-two (StringReader. "")
-          bootstrap-three (StringReader. "three\nfour")]
-      (is (=
-           ["one", "two", "three", "four"]
-           (chain-files [bootstrap-one bootstrap-two bootstrap-three]))))))
+(deftest parse-bootstrap-config-throws
+  (testing "throws error with line number and file"
```

Comment:
Might be worth explaining that this is testing loading a service that doesn't exist.